tribe-ext-use-newer-core-js/index.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Plugin Name:     The Events Calendar Extension: Use Newer Core JS
+ * Description:     Loads more current versions of core scripts like jQuery and Underscore when viewing various admin pages of The Events Calendar and Event Tickets.
+ * Version:         1.0.0
+ * Extension Class: Tribe__Extension__Use_Newer_Core_JS
+ * Author:          Modern Tribe, Inc.
+ * Author URI:      http://m.tri.be/1971
+ * License:         GPLv2 or later
+ * License URI:     https://www.gnu.org/licenses/gpl-2.0.html
+ */
+
+// Do not load unless Tribe Common is fully loaded.
+if ( ! class_exists( 'Tribe__Extension' ) ) {
+	return;
+}
+
+/**
+ * Extension main class, class begins loading on init() function.
+ */
+class Tribe__Extension__Use_Newer_Core_JS extends Tribe__Extension {
+
+
+	/**
+	 * Setup the Extension's properties. This always executes even if the required plugins are not present.
+	 *
+	 * @since 1.0.0
+	 */
+	public function construct() {
+		$this->add_required_plugin( 'Tribe__Events__Main' );
```

Comment:
I don't know off the top of my head, and it would unfortunately take quite a lot of digging to find a useful number here.

A couple of reasons why include the fact that there may be different min versions for the Underscore problem vs a jQuery problem, etc.

I would personally prefer and vote for omitting a minimum version number, because practically the only folks using this will be folks reporting a problem who are then redirected to this Extension to fix it.

Anyone _other_ than that crowd would get this Extension and not notice anything negative about it.